blog-server-api/src/endpoints/create_post/response_content_failure.rs

```diff
@@ -0,0 +1,52 @@
+use hyper::StatusCode;
+use screw_api::response::{ApiResponseContentBase, ApiResponseContentFailure};
+
+pub enum CreatePostContentFailure {
+    DatabaseError { reason: String },
+    ValidationError { reason: String },
+    AlreadyExists,
+    InsertFailed,
+}
+
+impl ApiResponseContentBase for CreatePostContentFailure {
+    fn status_code(&self) -> &'static StatusCode {
+        match self {
+            CreatePostContentFailure::DatabaseError { reason: _ } => {
+                &StatusCode::INTERNAL_SERVER_ERROR
+            }
+            CreatePostContentFailure::AlreadyExists => &StatusCode::BAD_REQUEST,
+            CreatePostContentFailure::ValidationError { reason: _ } => &StatusCode::BAD_REQUEST,
+            CreatePostContentFailure::InsertFailed => &StatusCode::INTERNAL_SERVER_ERROR,
+        }
+    }
+}
+
+impl ApiResponseContentFailure for CreatePostContentFailure {
+    fn identifier(&self) -> &'static str {
+        match self {
+            CreatePostContentFailure::DatabaseError { reason: _ } => "POST_DATABASE_ERROR",
+            CreatePostContentFailure::ValidationError { reason: _ } => "POST_VALIDATION_ERROR",
+            CreatePostContentFailure::AlreadyExists => "POST_ALREASY_EXISTS",
+            CreatePostContentFailure::InsertFailed => "COULD_NOT_FIND_CREATED_POST",
```

Comment:
🤡 prefix always related to endpoint name

Reply:
Changed